reconcile/utils/ocm.py

```diff
@@ -140,12 +147,60 @@ def create_cluster(self, name, cluster, dry_run):
             ocm_spec.setdefault('properties', {})
             ocm_spec['properties']['provision_shard_id'] = provision_shard_id
 
+        autoscale = cluster_spec.get('autoscale')
+        if autoscale is not None:
+            ocm_spec['nodes']['autoscale_compute'] = autoscale
+
         params = {}
         if dry_run:
             params['dryRun'] = 'true'
 
         self._post(api, ocm_spec, params)
 
+    def update_cluster(self, name, cluster, dry_run):
+        """
+        Updates a cluster.
+
+        :param name: name of the cluster
+        :param cluster: a dictionary representing a cluster desired state
+        :param dry_run: do not execute for real
+
+        :type name: string
+        :type cluster: dict
+        :type dry_run: bool
+        """
+        cluster_id = self.cluster_ids.get(name)
+        api = f'/api/clusters_mgmt/v1/clusters/{cluster_id}'
+        cluster_spec = cluster['spec']
+        ocm_spec = {
+            'nodes': {
+                'compute_machine_type': {
+                    'id': cluster_spec['instance_type']
+                }
+            },
+            'storage_quota': {
+                'value': float(cluster_spec['storage'] * pow(1024, 3))
```

Comment:
no need to calc `1024**3` all the time... let's use the actual number and add a commet... `... * 1073741824  # 1024 ** 3`

Reply:
makes sense